L1Trigger/Phase2L1ParticleFlow/interface/puppi/linpuppi_ref.h

```diff
@@ -35,6 +36,12 @@ namespace l1ct {
                      double priorNe,
                      double priorPh,
                      pt_t ptCut,
+                     bool useMLAssociation,
```

Comment:
Do we need sensible "defaults" for the case where we don't use the ML association?

Reply:
If not using ML association any values can be used for the ML association part.